src/cmd/manage_nsfs.js

```diff
@@ -239,7 +239,8 @@ async function verify_bucket_owner(bucket_owner, action) {
     }
     // check if bucket owner has the permission to create bucket (for bucket add only)
     if (action === ACTIONS.ADD && !account.allow_bucket_creation) {
-            throw_cli_error(ManageCLIError.BucketCreationNotAllowed, bucket_owner);
+            const detail_msg = `bucket owner ${bucket_owner}`;
+            throw_cli_error(ManageCLIError.BucketCreationNotAllowed, detail_msg);
```

Comment:
@shirady could you please add why bucket creation is not allowed for the owner in `BucketCreationNotAllowed`
This message does not give the actual real reason why bucket creation is not allowed for user. Something like
``` Account not allowed to create new buckets, Please add valid new_buckets_path and enable the flag allow_bucket_creation ``` (Assuming new_buckets_path needed)

Reply:
It is not part of the changes of this PR, but I will change it, thank you.

Reply:
@naveenpaul1 I noticed that changing the message fails some tests in the CI.
I would edit the detailed message instead.
Anyway, the original message is aligned with what we would encounter when using the NSFS server.

Reply:
@shirady This is a request I got from Ramya, May be we can add it in different PR

Reply:
The details will appear in the detail message as you can see in the latest commit.

Reply:
Attached output example:
```
  "error": {
    "code": "BucketCreationNotAllowed",
    "message": "Not allowed to create new buckets",
    "detail": "shira-1077 account not allowed to create new buckets. Please make sure to have a valid new_buckets_path and enable the flag allow_bucket_creation"
  }
```